src/components/ContentSidebar/Sidebar.js

```diff
@@ -27,6 +27,8 @@ type Props = {
     rootElement: HTMLElement,
     appElement: HTMLElement,
     onInteraction: Function,
+    onDescriptionChange: Function,
```

Comment:
`onDescriptionChange?: Function,`

Reply:
Why should this be optional? I'm always passing the function into sidebar.js. I also always pass it into detailsSidebar.js, and then it's conditionally passed to ItemProperties.

Reply:
I see